sdks/java/build-tools/src/main/resources/beam/suppressions.xml

```diff
@@ -21,6 +21,11 @@
   <suppress checks="JavadocPackage" files=".*[\\\/]maven-archetypes[\\\/].*"/>
   <suppress checks="JavadocPackage" files=".*[\\\/]examples[\\\/].*"/>
   
+  <suppress checks="JavadocMethod" files=".*Test\.java"/>
+  <suppress checks="JavadocMethod" files=".*[\\\/]src[\\\/]test[\\\/].*"/>
+  <suppress checks="JavadocMethod" files=".*[\\\/]testing[\\\/].*"/>
```

Comment:
I'm not sure about this one. If it is not in a `src/test` directory then it is not tests, but utility code that needs documentation.

Reply:
Good point. 

sending an update

Reply:
Done. Removed.   (apology after pulling from master and push again, the file status gets out of sync).